src/docfx/build/xref/ExternalXrefMapLoader.cs

```diff
@@ -13,22 +14,26 @@ namespace Microsoft.Docs.Build
     internal class ExternalXrefMapLoader
     {
         private static readonly byte[] s_uidBytes = Encoding.UTF8.GetBytes("uid");
+        private static readonly byte[] s_repositoryUrlBytes = Encoding.UTF8.GetBytes("repositoryUrl");
+        private static readonly byte[] s_docsetNameBytes = Encoding.UTF8.GetBytes("docsetName");
+        private static readonly byte[] s_referencesBytes = Encoding.UTF8.GetBytes("references");
+        private static readonly byte[] s_externalXrefsBytes = Encoding.UTF8.GetBytes("externalXrefs");
 
-        private static readonly byte[] s_repoUrlBytes = Encoding.UTF8.GetBytes("repositoryUrl");
-
-        public static IReadOnlyDictionary<string, Lazy<ExternalXrefSpec>> Load(Config config, FileResolver fileResolver, ErrorBuilder errors)
+        public static (IReadOnlyDictionary<string, Lazy<ExternalXrefSpec>> externalXrefMap, IReadOnlyList<Lazy<ExternalXref>> externalXref) Load(
```

Comment:
Make this return `ExternalXrefMap` immediately.
```suggestion
        public static ExternalXrefMap Load(
```

Reply:
OK.